### ISSUE TITLE
Add several post-translational modification relations under RO:0002436

### ISSUE THREAD

**Issue by cthoyt** [oborel/obo-relations#266]
As an addition to [RO:0002447 (phosphorylates)](http://purl.obolibrary.org/obo/RO_0002447) and [RO:0002480 (ubiquitinates)](http://purl.obolibrary.org/obo/RO_0002480), that occur under [RO:0002436 (molecularly interacts with)](http://purl.obolibrary.org/obo/RO_0002436), I would suggest adding relationships for the following post-translational modifications, which all have corresponding functions with the [Biological Expression Language (BEL)](https://biological-expression-language.github.io/) as well as corresponding Gene Ontology terms in the molecular function namespace:

| BEL Name | GO | GO Label |
| --- | --- | --- |
| chap | [GO:0044183](http://amigo.geneontology.org/amigo/term/GO:0044183) | protein binding involved in protein folding |
| gtp | [GO:0005525](http://amigo.geneontology.org/amigo/term/GO:0005525) | GTP binding |
| kin (already present) | [GO:0016301](http://amigo.geneontology.org/amigo/term/GO:0016301) | kinase activity |
| pep | [GO:0008233](http://amigo.geneontology.org/amigo/term/GO:0008233) | peptidase activity |
| phos | [GO:0016791](http://amigo.geneontology.org/amigo/term/GO:0016791) | phosphatase activity |
| ribo | [GO:0003956](http://amigo.geneontology.org/amigo/term/GO:0003956) | NAD(P)+-protein-arginine ADP-ribosyltransferase activity |
| tscript | [GO:0001071](http://amigo.geneontology.org/amigo/term/GO:0001071) | nucleic acid binding transcription factor activity |
| tport | [GO:0005215](http://amigo.geneontology.org/amigo/term/GO:0005215) | transporter activity |
| gef | [GO:0005085](http://amigo.geneontology.org/amigo/term/GO:0005085) | guanyl-nucleotide exchange factor activity |
| gap | [GO:0032794](http://amigo.geneontology.org/amigo/term/GO:0032794) | GTPase activating protein binding |

There are also several other entries in [cellular protein modification process](http://amigo.geneontology.org/amigo/term/GO:0006464) (e.g., amination, nitration, polyamination, nitrosylation, neddylation, sumoylation, etc.) but let's just start by thinking about a couple concrete ones :)

I think many of these make sense following the form proposed by @cmungall (see: https://github.com/oborel/obo-relations/issues/259#issuecomment-424472190) with the addition of `mf` to represent one of these molecular functions that are a [cellular protein modification process](http://amigo.geneontology.org/amigo/term/GO:0006464):

```
?a1 enabled_by ?gp1
?gp1 a ?mf
?mf a GO:'cellular protein modification process'
?a1 directly-regulates ?a2
?a2 enabled-by ?gp2
->
?gp1 XXX ?gp2
```

One caveat: a couple of these aren't "activities" but come from the "binding" tree. I'm also curious how one would handle those.

**Comment by nlharris** [comment]
Are these still needed?

**Comment by cthoyt** [comment]
Yes I'm still interested in adding additional relationships (and reverse relationships). I'll update the issue header with some more information

**Comment by nlharris** [comment]
Is this now done?

**Comment by cthoyt** [comment]
It appears there are some issues with the build and these new terms are not being propagated. Unfortunately, of the issues/PRs/commits that were a follow-up to #521 are very cryptic:

- #586
- #587
- c2157c9f6a9e885bfe619196bee298066ec77edd

The new termsin the `ro-edit.owl` don't appear to be in the release files. e.g., a search in https://github.com/oborel/obo-relations/blob/master/ro.obo for `myristolates` currently shows nothing. Does this also means there's an overall problem with the build?

**Comment by matentzn** [comment]
cc @anitacaron

**Comment by anitacaron** [comment]
@cthoyt I checked the IDs created in PR #586 and I could found in the last RO release. Try to search for `myristoylates`.

![Screenshot 2022-11-07 at 09 32 08](https://user-images.githubusercontent.com/2208124/200276693-b8100fdd-c82a-433e-a0a5-bf8ca86269d0.png)

**Comment by cthoyt** [comment]
Yes, but that one was done by hand - I mean the remaining ones that we curated in https://github.com/oborel/obo-relations/blob/master/src/patterns/ro-interaction-relations-src.yaml to be generated via template are not getting propogated

**Comment by anitacaron** [comment]
Yes, they need to be moved to the components folder properly.

**Comment by anitacaron** [comment]
@cthoyt I need to double-check if `linkml-owl` was added to ODK so it could be easily generated in the `ro.Makefile`. The current `patterns` folder is used for the DOSDP.
